src/test/java/uk/gov/hmcts/reform/civil/utils/AccessCodeGeneratorTest.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.civil.utils;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static uk.gov.hmcts.reform.civil.utils.AccessCodeGenerator.generateAccessCode;
+
+public class AccessCodeGeneratorTest {
+
+    @Test
+    void shouldReturnNull_whenAllValuesPassedAreNull() {
```

Comment:
shouldReturnNull_whenAllValuesPassedAreNull this method does not do what the name is saying.

Reply:
Changed the method name to a proper ones